portality/constants.py

```diff
@@ -38,8 +38,8 @@
 IDENT_TYPE_DOI = "doi"
 LINK_TYPE_FULLTEXT = "fulltext"
 
-TODO_MANED_STALLED = "todo_maned_stalled"
-TODO_MANED_FOLLOW_UP_OLD = "todo_maned_follow_up_old"
-TODO_MANED_READY = "todo_maned_ready"
-TODO_MANED_COMPLETED = "todo_maned_completed"
-TODO_MANED_ASSIGN_PENDING = "todo_maned_assign_pending"
+TODO_MANED_STALLED = "Chase <strong>stalled</strong> application"
+TODO_MANED_FOLLOW_UP_OLD = "Follow up with <strong>old<strong> application"
+TODO_MANED_READY = "Make a decision on <strong>ready<strong> application"
+TODO_MANED_COMPLETED = "Review <strong>completed<strong> application"
+TODO_MANED_ASSIGN_PENDING = "Assign <strong>pending<strong> application"
```

Comment:
A bit of a misunderstanding here for the meaning of this file.  `constants` is for internal references within the code to the same thing, so that if we change the value of the variable, everything still hangs together.  It isn't "safe" to use this for display, we'd need to define a mapping from those values to the display value somewhere else (probably `ui.messages`).  

I complicated matters by displaying it, but that was only because there wasn't a UI for the todo list yet :)

Reply:
I'm pushing a change to show what I mean

Reply:
No problem! I had originally done it that way but then I saw that some of the variables had the "human-readable" names (like `INDEX_RECORD_TYPE_UPDATE_REQUEST_UNFINISHED`), so I thought perhaps the template approach was wrong! 

Thanks for fixing.  

Is there anything else I should do, besides correcting the two conflicts?

Reply:
Is there anything else you needed me to do here?